holidays/holiday_base.py

```
@@ -809,6 +809,54 @@ def _add_holiday(self, name: str, *args) -> date | None:
         self[dt] = self.tr(name)
         return dt
 
+    def _add_multiday_holiday(self, *args) -> set[date]:
```

Ruff (0.14.4)

Missing type annotation for `*args`

(ANN002)